File: SocialBackend/src/main/java/com/collaborate/SocialBackend/service/JobService.java
package com.collaborate.SocialBackend.service;


import java.util.List;

import com.collaborate.SocialBackend.model.Job;

public interface JobService {
    List<Job> getAllJobs();
    
    public boolean updatejob(Job job);
    
    public boolean deletejob(int jobId);
    
    public boolean addjob(Job job);
    
    public Job getjobById(int  jobId);
    
    
        
}

File: SocialFrontend/WebContent/services/jobservice.js
/**
 * 
 */
app.factory('JobService',function($http){
	var jobService={}
	var BASE_URL="http://localhost:8080/SocialMiddleware"
		jobService.addjob=function(job){
		return $http.post(BASE_URL+"/addjob",job)
	}
	jobService.getAllJobs=function()
	{
		return $http.get(BASE_URL+"/getallJobs")
	}
	return jobService;
})  
